alternator/executor.cc

```diff
@@ -105,6 +105,21 @@ static const column_definition& attrs_column(const schema& schema) {
     return *cdef;
 }
 
+
+static std::shared_ptr<stats> get_stats_from_schema(service::storage_proxy& sp, const schema& schema) {
+    try {
+        replica::table& table = sp.local_db().find_column_family(schema.id());
+        if (! table.get_stats().alternator_stats) {
+            table.get_stats().alternator_stats = std::make_shared<table_stats>(schema.ks_name(), schema.cf_name());
+        }
+        return table.get_stats().alternator_stats->_stats;
+    } catch (std::runtime_error&) {
```

Comment:
find_column_family() throws no_such_column_family when the table is gone. Any reason why you decided to catch a more general exception? I think it's not a good idea. If we have a different error (out of memory? IO error? I don't know what?) I think we need to fail, we shouldn't just ignore it.

Reply:
this part is only a safe way of getting a stats object, if it fails here, it would fail during the operation, so I let the regular fail path return the write error code, no need to handle another exception everywhere just for the stats object